library/templates/nycoc_checkbook.yml

```diff
@@ -0,0 +1,49 @@
+dataset:
+  name: &name nycoc_checkbook
+  version: "{{ version }}"
+  acl: private 
```

Comment:
since the data isn't confidential/sensitive, let's make this `public` so our loading of it later can be relatively simple